AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponentsLevel_DiffuseGlobalIlluminationAdded.py

```diff
@@ -86,10 +86,10 @@ def AtomEditorComponentsLevel_DiffuseGlobalIllumination_AddedToEntity():
 
         # 4. Set Quality Level property to Low
         diffuse_global_illumination_component.set_component_property_value(
-            AtomComponentProperties.diffuse_global_illumination('Quality Level', GLOBAL_ILLUMINATION_QUALITY['Low']))
+            AtomComponentProperties.diffuse_global_illumination('Quality Level'), GLOBAL_ILLUMINATION_QUALITY['Low'])
         quality = diffuse_global_illumination_component.get_component_property_value(
             AtomComponentProperties.diffuse_global_illumination('Quality Level'))
-        Report.result(diffuse_global_illumination_quality, quality == GLOBAL_ILLUMINATION_QUALITY['Low'])
+        Report.result(Tests.diffuse_global_illumination_quality, quality == GLOBAL_ILLUMINATION_QUALITY['Low'])
```

Comment:
Why did Tests needed to be added here? I didn't see any import changes....

Reply:
This had to reference the "Tests" portion of the file. With out it, nothing could be reported and the test would fail.